### PR TITLE
Some more install information

### PR DESCRIPTION
**Pull request by psonnera**
That's a botched one and some pictures need to be replaced as I didn't have a Libre streaming but that's the idea. Let me know if it helps, maybe you can pick ideas from it. Don't merge it like that as it collides with your readme.
Thanks for the app!

### CODE REVIEW

**Comment by pachi81** [comment]
Hey!
Thank you very much for your input!!!
That helps a lot and save me some time!
At the moment I have more focus on the app itself and know, that the docu has to be changed...
I also thought about adding a screenshot of each complication.

File: install.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+</br>
+
+## 2 - Install GlucoDataHandler on your watch
+
+Now download `GlucoDataHandler-Wear.apk` on your phone but **do not install it**.
+
+<img src='images/download.png' width=350 align=left>
+
+### Method #1 - Install with Wear Installer 2
```
Comment (pachi81) [inline install.md:46]: Just a question, does it is working for you?
I have tried Wear Installer 1 and 2, but both apps are showing always success, even if the installation fails. Except of one time, it haven't installed anything...

Comment (psonnera) [reply install.md:46]: No, it failed with both when I tried... but a few days ago I help a guy in the UK and he managed to install with Wear Installer... Same pre-release... I used ADB that's the most reliable way I know.

**Comment by pachi81** [comment]
Hi, it will take some more time, because yesterday I was sick, today I have a lot of work todo and tomorrow I´m going in ski holiday for a week. So I can do this change after my holiday.

**Comment by pachi81** [comment]
Merged it after doing some changes...

**Pull request closed without merging**